Security/TwoFactor/AuthenticationContext.php

```diff
@@ -108,4 +108,12 @@ public function setAuthenticated($authenticated)
     {
         $this->authenticated = $authenticated;
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function isAuthenticationTry()
+    {
+        return $this->request->request->has('_auth_code');
+    }
```

Comment:
Done but _auth_code isn't dynamic in [Authentication/form.html.twig](https://github.com/scheb/two-factor-bundle/blob/master/Resources/views/Authentication/form.html.twig#L7) in input field name.
I will create a PR after this one